COLLADAStreamWriter/include/COLLADASWConstants.h

```diff
@@ -202,6 +202,8 @@ namespace COLLADASW
         static const String CSW_ELEMENT_PASS;
         static const String CSW_ELEMENT_PH;
         static const String CSW_ELEMENT_PHONG;
+		static const String CSW_ELEMENT_PLUGIN_VERSION;
```

Comment:
what is this constant ?
I see CURRENT_REVISION and CURRENT_SHA1 defined above, but I am not sure what this contains?